src/Database/PostUpdate.php

```diff
@@ -120,7 +120,8 @@ private static function update1198()
 		logger("Start", LOGGER_DEBUG);
 
 		// Check if the first step is done (Setting "author-id" and "owner-id" in the item table)
-		$r = dba::select('item', ['author-link', 'owner-link', 'uid'], ['author-id' => 0, 'owner-id' => 0], ['limit' => 1000]);
+		$fields = ['author-link', 'author-name', 'author-avatar', 'owner-link', 'owner-name', 'owner-avatar', 'network', 'uid'];
```

Comment:
Having an extra local variable is really not needed here. Maybe put back the array in the `dba::select()` command's parameter list? And then reformat it to have it more clear?

Reply:
We are using this style with the variable ```$fields``` at many places and I like it, since it leaves the ```dba::``` more easily to read than some strange multi line creature.

Reply:
As a rule of thumb, if the dba call is longer than 120 chars, I use secondary variables for readability.

Reply:
Okay.